test/test_project_insights.py

```diff
@@ -96,7 +96,7 @@ async def test_project_insights_summary(self):
         expected = [
             ProjectInsightsSummaryInternal(
                 project=self.project_id,
-                dataset='test',
+                dataset=self.project_name,  # for ProjectInsights, dataset is project.name
```

Comment:
This was a bit unexpected to my eyes, but this is how `dataset` is initialised in _db/python/layers/project_insights.py_…